{"frames":[{"func":"plot(data2)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in plot(data2) : object 'data2' not found\n"}